UI/MVUX/src/global.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
@Kunal22shah can you add the MVUX sample app to this list at the same time please:
https://github.com/unoplatform/Uno.Samples/blob/master/doc/samples.md